tesseract_utils.py

```diff
@@ -1,19 +1,21 @@
-import numpy
-
 from classification import classes_from_numpy, classes_to_numpy
 from segmentation import segments_from_numpy, segments_to_numpy
+import sys
```

Comment:
This is also with purpose. I'm overriding the builtin because I have to pass an `encoding` kwarg. The `io` module is the Python 3 io framework, but backporten to Py2. It is slower though, and thus not preferred on Python 3.

It could be changed to a version check and then a version with and one without the encoding kwarg passed, if you prefer that.